### PR TITLE
Pg samples

### PR DESCRIPTION
**Pull request by olavloite**
Adds samples and integration tests for PG features. These samples should be added to the `golang-samples` repository, but as the PG features are not available over there, the samples are temporarily parked here. The samples and integration tests can be copied without any changes to the `golang-samples` repository. The additional test-util and infrastructure code should not be copied, as this is already present in `golang-samples`.

### CODE REVIEW

**Comment by olavloite** [comment]
cc @ansh0l

File: spanner/golang-samples/spanner/pg_spanner_connect_to_database.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+import (
+	"context"
+	"io"
+
+	"cloud.google.com/go/spanner"
+	database "cloud.google.com/go/spanner/admin/database/apiv1"
+)
+
+func createPgClients(w io.Writer, db string) error {
```
Comment (rahul2393) [inline spanner/golang-samples/spanner/pg_spanner_connect_to_database.go]: Could you please add comments for these functions: https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#comment-functions-and-packages

Comment (rahul2393) [reply spanner/golang-samples/spanner/pg_spanner_connect_to_database.go]: Please provide example values for functions: https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#function-arguments-for-snippets

Comment (olavloite) [reply spanner/golang-samples/spanner/pg_spanner_connect_to_database.go]: Done

**Comment by olavloite** [comment]
cc @jin-jj

**Review by rahul2393 — APPROVED** [review]
LGTM

**Pull request merged**